From 6b0f3f2835f756bf5b8adfb850e8f379af42a9f6 Mon Sep 17 00:00:00 2001
From: William Woodall <william@osrfoundation.org>
Date: Wed, 2 Mar 2016 16:07:14 -0800
Subject: [PATCH] add example change

---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 401540e..8719f1f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,6 +8,8 @@ if (POLICY CMP0054)
   cmake_policy(SET CMP0054 NEW)
 endif()
 
+message("Hello!")
+
 find_package(Boost REQUIRED
   COMPONENTS
   filesystem
-- 
2.5.4 (Apple Git-61)

